packages/litexa/src/parser/skill.coffee

```diff
@@ -394,8 +393,11 @@ class lib.Skill
     do =>
       output.push "litexa.dbTypes = {"
       lines = []
-      for name, def of @dbTypes
-        lines.push "  #{name}: { type: '#{def.type}' }"
+      dbTypeSet = {}
+      for dbType in @dbTypes
+        if dbType.name not of dbTypeSet
+          lines.push "  #{dbType.name}: { type: '#{dbType.type}' }"
+          dbTypeSet[dbType.name] = dbType
```

Comment:
This doesn't look right: This would basically override any localized DB type definitions with the default DB type definitions.